tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -47,6 +47,9 @@ EOF
 	ln -s /run "$rootfs_dir/var/run"
 	cp --remove-destination /etc/resolv.conf "$rootfs_dir/etc"
 
+	mkdir -p "$rootfs_dir/etc/ssl/certs"
+	cp --remove-destination /etc/ssl/certs/ca-certificates.crt "$rootfs_dir/etc/ssl/certs"
```

Comment:
Nit: Duplicated directory.

```suggestion
        local dir="$rootfs_dir/etc/ssl/certs"
	mkdir -p "$dir"
	cp --remove-destination /etc/ssl/certs/ca-certificates.crt "$dir"
```

Reply:
Fixed.